spec/features/global_announcements_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Global announcements', type: :feature do
+  subject { page }
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    describe 'no global announcements' do
+      before do
+        instance.announcements.clear
```

Comment:
The suggested solution does not address my concern that all the tests now depend on an empty set of instance announcements. Adding another test now will have that test start with an empty set of announcements, which is stateful even if the test does not require that precondition.

Your solution to that is to make a new file. I think that is also a bad idea because now the specs for one feature is split over two places.

You might not think that this is important, but I do. I've seen projects grow and I've seen code bases evolve. There's always unquantifiable changes that happen to code bases over time and I'm trying to make sure as few ways these things can happen. If you think that your suggestion is still better in the long run, I'll leave the discussion right here and adopt your solution.